cdap-watchdog/src/test/java/co/cask/cdap/logging/framework/CDAPLogAppenderTest.java

```diff
@@ -213,6 +223,14 @@ public void testCDAPLogAppenderRotation() throws Exception {
       Assert.assertEquals(files.get(1).getEventTimeMs(), currentTimeMillisEvent1 + 1000);
       Assert.assertTrue(files.get(0).getFileCreationTimeMs() >= currentTimeMillisEvent1);
       Assert.assertTrue(files.get(1).getFileCreationTimeMs() >= currentTimeMillisEvent2);
+
+      // checking permission
+      String expectedPermissions = "rwxr-----";
```

Comment:
so i added a comment as to why i do this. the permission is configurable, also in mac if i have 600 permission (owner has only read-write but not execute) the test will fail to create sub-dir. so i configured the test case property to have "rwx" for the owner.